src/pages/index.js

```diff
@@ -90,44 +90,93 @@ const profileEditPopup = new PopupWithForm(
 profileEditPopup.setEventListeners();
 
 // Profile Edit Popup Button
+const profileEditButton = document.querySelector("#profile-edit-button");
+
 profileEditButton.addEventListener("click", () => {
-  const userInfoData = userInfo.getUserInfo();
+  if (userInfoInstance) {
+    const userInfoData = userInfoInstance.getUserInfo();
 
-  profileTitleInput.value = userInfoData.name;
-  profileDescriptionInput.value = userInfoData.job;
+    profileTitleInput.value = userInfoData.name;
+    profileDescriptionInput.value = userInfoData.job;
 
-  profileEditPopup.open();
+    profileFormValidator.resetValidation();
+    profileEditPopup.open();
+  }
 });
 
-// Profile Changes Save
 function handleProfileSaveSubmit(inputValues) {
-  userInfo.setUserInfo({
+  const saveButton = document.getElementById("profile-modal-save-button");
+
+  const newInfo = {
     name: inputValues.title,
-    job: inputValues.description,
-  });
+    about: inputValues.description,
+    avatar: avatarImg.src,
```

Comment:
`    avatar: avatarImg.src,
`

You don't need this because you can't take the data from `DOM` elements like that anymore. You have `UserInfo` for getting such data. But... you receive the whole data in the response from the server. You have avatar there as well


Please, use responses